src/http.cc

```diff
@@ -771,10 +771,9 @@ HttpStateData::handle1xx(const HttpReply::Pointer &reply)
     // check whether the 1xx response forwarding is allowed by squid.conf
     if (Config.accessList.reply) {
         ACLFilledChecklist ch(Config.accessList.reply, originalRequest().getRaw());
-        ch.al = fwd->al;
-        ch.reply = reply.getRaw();
+        ch.updateAle(fwd->al);
+        ch.updateReply(reply);
```

Comment:
In this and similar PR contexts, it is possible that ALE::reply exists and, hence, calling updateReply() is not necessary. However, investigating these hidden dependencies between ALE::reply and job reply fields requires significant effort (especially where a single job deals with multiple reply objects). I did not perform that investigation for this PR, preserving the original "update both ALE and reply" logic (except for trivial cases where the reply object was obviously taken from the ALE object in the same code snippet). I believe that investigation and related further improvements should be viewed as out of scope.